src/Sylius/Bundle/AdminBundle/Command/ChangeAdminUserPasswordCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Command;
+
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Sylius\Component\User\Security\PasswordUpdaterInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'sylius:admin-user:change-password',
+    description: 'Change password of admin user'
+)]
+final class ChangeAdminUserPasswordCommand extends Command
+{
+    private SymfonyStyle $io;
+
+    public function __construct(
+        private UserRepositoryInterface $adminUserRepository,
+        private PasswordUpdaterInterface $passwordUpdater
+    ) {
+        parent::__construct();
```

Comment:
Not needed

Reply:
@Zales0123 what do you mean? The call to `parent::__construct()` is not needed or should I remove the whole constructor?

Reply:
This is because the parent constructor initializes  InputDefinition and Aliases ...

Reply:
Yup, once the command's constructor override, it must call the parent's one.